xAPI-About.md

```diff
@@ -396,9 +397,12 @@ having a "home email" and a "work email".  Both are the same person, but have di
 
 <a name="def-profile" ></a>
 
-__Profile__: A construct where information about the learner or activity is kept, typically in name/document pairs 
-that have meaning to an instructional system component.  Note that "profile" is also used to describe specifications
-building on xAPI that Communies of Practice may develop.
+__Profile__: A profile is a specific set of rules and documentation for implementing xAPI in a particular context. Profiles generally provide a particular vocabulary of terms, some created specifically for the profile, and some are referenced from other vocabularies. Sometimes a profile might provide multiple vocabularies for different situations, and sometimes someone might curate a vocabulary from multiple sources without creating a profile.
```

Comment:
Stylistically, I think it's preferable if the definitions omit the "A profile is..." repetition of the thing being defined. Currently, the "Definitions" section is inconsistent in this regard. This seems like a good time to add consistency.